src/consensus/aft/raft.h

```diff
@@ -2406,7 +2438,6 @@ namespace aft
 
       LOG_INFO_FMT(
```

Comment:
Can we add a test for this in `reconfiguration.py:test_retire_primary`? We should be able to observe that the commit level on majority of backups has gone past the `seqno` at which the retire node proposal was accepted before the election is started.

Reply:
I have no idea how to do this without a serious race condition I'm afraid, do you have something specific in mind?

Reply:
Fair enough. The election may have already happened by the time we check for commit of the primary's retirement.